media-video/gxine/gxine-0.5.909.ebuild

```diff
@@ -13,27 +13,32 @@ SLOT="0"
 KEYWORDS="amd64 ppc ppc64 x86"
 IUSE="l10n_cs l10n_de lirc nls nsplugin udev xinerama"
 
-COMMON_DEPEND=">=media-libs/xine-lib-1.1.20[gtk]
+DEPEND="
+	media-libs/xine-lib[gtk]
 	x11-libs/gtk+:2
-	>=dev-libs/glib-2
-	>=dev-lang/spidermonkey-1.8.2.15:0
+	dev-lang/spidermonkey:0
```

Comment:
We shouldn't change too many style things for other developers' ebuilds but if it's a "quiet project", go ahead. We should sort these deps.

Reply:
Agreed, though most of these ebuilds are not regularly maintained or haven't been touched in long time so I think we're safe